lib/opensusebasetest.pm

```diff
@@ -293,6 +293,25 @@ sub rewrite_static_svirt_network_configuration {
     type_line_svirt "systemctl is-active network",             expect => 'active';
 }
 
+=head2 wait_for_support_server
+
+  wait_for_support_server():
+
+Wait for the support server to finish its initialization.
+Ideally, this should be done *before* starting the OS, mainly if a DHCP
+server is needed.
+
+=cut
+sub wait_for_support_server {
+    return unless get_var('USE_SUPPORT_SERVER');
```

Comment:
I think returning at first line of function makes it a bit hard to debug code and reuse function. I would prefer
`wait_for_support_server if get_var('USE_SUPPORT_SERVER');`

Feel free to ignore comment if you don't agree, just an opinion..

Reply:
same as comment above just opinion which you may ignore - what about shorter name - `wait_supportserver` ?

Reply:
Your comment is good, I don't know what is better honestly.

What are the others opinion? I will follow the majority :)

Reply:
Ok for short name, and it seems that most people prefer to use the `wait_supportserver if get_var('USE_SUPPORT_SERVER');` form :)

So I change this!